magic/src/main/scala/com/fortysevendeg/scalacheck/magic/Magic.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * scalacheck-datetime
+ * Copyright (C) 2016-2017 47 Degrees, LLC. <http://www.47deg.com>
+ */
+
+package com.fortysevendeg.scalacheck.magic
+
+import org.scalacheck._
+import org.scalacheck.Gen._
+import org.scalacheck.Arbitrary.arbitrary
+
+object Magic {
+
+  private[magic] val strings = List("This", "is", "a", "temporary", "list", "of", "strings")
+
+  val magicStrings: Gen[String] = oneOf(strings)
+
+  implicit val enhancedStringArbitrary: Arbitrary[String] = Arbitrary {
```

Comment:
I wonder if for cases like this one where we are explicitly overriding the scalacheck default instance for `Arbitrary[String]` an alternative approach would be to use tagged types so it's explicit that we are summoning the implicit for magic Strings. `Arbitrary[String @@ Magic]`